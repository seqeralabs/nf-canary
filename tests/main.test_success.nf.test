nextflow_process {

    name "Test Process TEST_SUCCESS"
    script "main.nf"
    process "TEST_SUCCESS"

    test("Should run without failures") {

        then {
            assert process.success
            assert process.exitStatus == 0
            assert process.trace.tasks().size() == 1
            assert snapshot(process).match()
        }

    }

}
