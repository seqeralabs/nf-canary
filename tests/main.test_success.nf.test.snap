{
    "Should run without failures": {
        "content": [
            {
                "stderr": [
                    
                ],
                "errorReport": null,
                "exitStatus": 0,
                "out": {
                    "0": [
                        ""
                    ]
                },
                "failed": false,
                "stdout": [
                    
                ],
                "errorMessage": null,
                "trace": {
                    "tasksFailed": 0,
                    "tasksCount": 1,
                    "tasksSucceeded": 1
                },
                "name": "TEST_SUCCESS",
                "success": true
            }
        ],
        "timestamp": "2023-10-04T12:20:24.512086"
    }
}